tests/unit/core/metrics/test_confusion_matrix.py

```diff
@@ -10,6 +12,36 @@
 from whylogs.proto import ScoreMatrixMessage
 
 
+def _generateTestTargetsAndPredictions(label_size: int, output_length, rand_ratio: float):
+    labels = [i for i in range(label_size)]
```

Comment:
why not just `labels = range(label_size)` ?

Reply:
yeah better to simplify, done.